test/cli/utils_test.go

```diff
@@ -225,7 +225,11 @@ func getSyftBinaryLocationByOS(t testing.TB, goOS string) string {
 	// note: there is a subtle - vs _ difference between these versions
 	switch goOS {
 	case "darwin", "linux":
-		return path.Join(repoRoot(t), fmt.Sprintf("snapshot/%s-build_%s_%s/syft", goOS, goOS, runtime.GOARCH))
+		archPath := runtime.GOARCH
```

Comment:
nit: i feel it would be better to put the arch addition before the switch case in case we add more cases to the switch case in the future.